lib/ansible/modules/system/ufw.py

```diff
@@ -303,13 +319,47 @@ def ufw_version():
         if command == 'state':
             states = {'enabled': 'enable', 'disabled': 'disable',
                       'reloaded': 'reload', 'reset': 'reset'}
-            execute(cmd + [['-f'], [states[value]]])
```

Comment:
I noticed that there seem to be two bugs in `ufw`:
 - `ufw -f reset` does not disable the firewall (as advertised, status is `active` afterwards);
 - `ufw --dry-run -f reset` ignores the `--dry-run` flag (and clears all rules).

Reply:
(So handling check mode differently will actually improve the module :) )

Reply:
https://bugs.launchpad.net/ufw/+bug/1810082

Reply:
BTW, there is another `--dry-run` bug (for `enable` and `disable`): https://bugs.launchpad.net/ufw/+bug/1808727